Zwave Climate: Fan state attribute missing (#25287)

jaminh opened the pull request:
**Related issue (if applicable):** fixes #25287

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [ ] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [ ] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [ ] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [ ] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

homeassistant commented:
Hi @jaminh, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant#25573).

Once you do that we will be able to review and accept this pull request.

Thanks!

Adminiuga reviewed (dismissed):
Looks good

MartinHjelmare commented on homeassistant/components/zwave/climate.py:
```diff
@@ -256,6 +256,11 @@ def hvac_action(self):
         return self._hvac_action
 
+    @property
+    def fan_state(self):
```
Where is this used?

MartinHjelmare replied:
I think we can remove this property and use `self._fan_state` directly in the state attributes.

pvizeli commented:
That fix nothing because there is no code that read this property.

pvizeli reviewed (changes requested):
Please provide screenshots with before/after

jaminh committed:
Zwave Climate: Fan state attribute missing (#25287)

Add fan state back in to device state attributes
Change fan state to fan action

jaminh commented:
Fan state is back with latest changes.
![Screenshot from 2019-07-30 08-34-33](https://user-images.githubusercontent.com/3012496/62135388-bfe13200-b2a7-11e9-827f-4302b66ec4b3.png)

pvizeli commented on homeassistant/components/zwave/climate.py:
```diff
@@ -289,3 +289,11 @@ def set_swing_mode(self, swing_mode):
+        """Return the optional state attributes."""
+        data = super().device_state_attributes
+        if self.fan_state:
+            data[ATTR_FAN_STATE] = self.fan_state
```
call this `fan_action`, could be that we add this later like the `hvac_actions`

jaminh replied:
I changed all references from fan "state" to "action" and I was able to see the new attribute in my install so I think everything should be working. I can squash all the commits down to one unless there are more changes?
